Makefile

```diff
@@ -24,7 +24,7 @@ format:
 	@gofmt -l  -w ./
 
 check: test
-	@test -z $(shell gofmt -l ./ | tee /dev/stderr) || (echo "[WARN] Fix formatting issues with 'make fmt'"; exit 1)
```

Comment:
Probably this commit should be extracted into the separate PR?

Reply:
ok